src/common.cpp

```diff
@@ -1898,6 +1901,63 @@ std::string get_executable_path(const char *argv0) {
         }
         return buffstr;
     }
+#else
+    static std::string result;
+    if (!result.empty()) return result;
+    std::string strargv0 = ((argv0) ? argv0 : ""), buffstr;
```

Comment:
Please don't define two variables on the same line.

I had trouble figuring out where `buffstr` was defined.